_config.yml

```diff
@@ -97,4 +103,5 @@ defaults:
       path: "ca"
     values:
       lang: "ca"
-      label: "Català"
\ No newline at end of file
+      label: "Català"
+theme: jekyll-theme-minimal
```

Comment:
Stray line?

Reply:
Sorry, I do not remenber, why added the theme.  May be I didn't like the site look in my browser.